src/server/game/Entities/Creature/Creature.h

```diff
@@ -659,7 +659,7 @@ class Creature : public Unit, public GridObject<Creature>, public MapObject
 
         // Handling caster facing during spellcast
         void SetTarget(ObjectGuid guid) override;
-        void FocusTarget(Spell const* focusSpell, WorldObject const* target);
+        void FocusTarget(Spell const* focusSpell, ObjectGuid targetGuid);
```

Comment:
Some creatures stop targetting while casting example: http://www.wowhead.com/npc=72762 when casting http://www.wowhead.com/spell=147310/gust-of-wind